src/popup/components/TransactionOverview.vue

```diff
@@ -1,10 +1,8 @@
 <template>
   <TransactionInfo
     class="transaction-overview"
-    :title="preparedTransaction.title"
-    :sender="preparedTransaction.sender"
-    :recipient="preparedTransaction.recipient"
-    :transaction-function="preparedTransaction.function"
+    :sender="transactionParties.sender"
+    :recipient="transactionParties.recipient"
```

Comment:
These 2 properties and `transaction` are the only ones that `TransactionInfo` component can receive. The rest was unused and there was no need to calculate them.